pkg/controller/operator.go

```diff
@@ -112,6 +116,36 @@ func (s *Support) Run(controller *controllercmd.ControllerContext) error {
 	authorizer := clusterauthorizer.New(configObserver)
 	insightsClient := insightsclient.New(nil, 0, "default", authorizer, configPeriodic)
 
+	// read proxy status
+	insightsClient.GetProxyStatusFunc = func() (*configv1.ProxyStatus, error) {
+		proxyConfig, err := gatherConfigClient.Proxies().Get("cluster", metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			return nil, nil
+		}
+		if err != nil {
+			return nil, err
+		}
+		return &proxyConfig.Status, nil
+	}
+
+	// read proxy CA bundle
+	insightsClient.GetProxyRootCAsFunc = func() (*x509.CertPool, error) {
```

Comment:
Should probably be `GetTrustedCABundleFunc`, the only connection `trusted-ca-bundle` has to proxy is that it may contain CA cert for a TLS MITM proxy, but otherwise it's really quite a general trust bundle